bootstrapvz/common/tasks/cleanup.py

```diff
@@ -30,3 +30,15 @@ def run(cls, info):
         log = os.path.join(info.root, 'var/log/')
         os.remove(os.path.join(log, 'bootstrap.log'))
         os.remove(os.path.join(log, 'dpkg.log'))
+
+
+class ClearMACHINEID(Task):
```

Comment:
Heh, no need for all uppercase. It's just that MOTD and TMP are actually acronyms. I suggest `ClearMachineId`.

Reply:
OK